db/migrate/20210318204533_add_import_type_to_import_histories.rb

```diff
@@ -0,0 +1,15 @@
+class AddImportTypeToImportHistories < ActiveRecord::Migration[6.0]
+  def change
+    add_column :katello_content_view_version_import_histories,
+               :import_type, :string,
+               :default => ::Katello::ContentViewVersionExportHistory::COMPLETE, :null => false
+    add_index :katello_content_view_version_import_histories, :import_type, name: 'katello_cvveh_import_type'
+
+    ::Katello::ContentViewVersionImportHistory.reset_column_information
```

Comment:
The import histories just landed on master (not in any release)  so I don't think this part of the code which sets the import_type will ever be executed

Reply:
Sure its not in any release but I was thinking about dev exports that might happen between mine and jeremy's prs . 
As a general rule when I add a new migration if I had previous records I prefer updating here.

Reply:
disagree?

Reply:
Totally agree with that as a general rule. I think this could be an exception though. It's only you and jeremy with these records since the code was just merged yesterday and doubt that your env is gonna suffer for not having import_type set. Otherwise this code is gonna be in the app forever!